lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -75,6 +75,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
     'crop_width' => $values['crop_width'],
     'crop_height' => $values['crop_height'],
     'crop_rotate' => $values['crop_rotate'],
+    'source' => isset($values['source']) ? $values['source'] : NULL,
```

Comment:
Should we start sending a source value to rogue like `phoenix_web` or something for when a reportback is sent from the reportback form to rogue? I know we don't usually do it for the web form but when rogue starts collecting reportbacks from different sources that could be a nice to have. Thoughts?